### PR TITLE
Added work and about section to the portifolio

### PR DESCRIPTION
**Pull request by Etsubdink-Demeke**
On this pull request, I added the work section and about section to the portfolio. 
-I added flexbox and grid layout.
-fix the background of the button
-change the background color of  the body
-changed the layout of tech-stack 
-fix everything in the last pull request

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,5 +36,190 @@
...
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+          </span>
+        <div class="button">
+          <button>see projects</button>
+        </div>
+      </div>
+      </div>
```
Comment (Kamba56) [inline index.html]: - [ ] As demanded by project requirements, you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly consider the following;
  - Edit the background color you have on the body to match the one on the figma template.
  - Add a border radius to each work card to match the template
  - Add a border to work card to match template
  - Kindly edit the list of technologies layout and colors for each work card to match template
image
  -  Kindly edit this section layout to match design template
image
  - Kindly edit the background color and border of your buttons to match design template
  - Kindly add some space between work cards to match template.
  - I noticed your work card isn't centralized. The space on the left is greater than that on the right. Kindly fix this please.

### Your design
image
### Figma
image

File: index.html [hunk 2]
```diff
@@ -36,5 +36,190 @@
...
+    
+    <div class="cards">
+      
+      <div class="screenshoot">
+        <img src="images/Snapshoot Portfolio (4).png" alt="portifolio screenshoot">
+      </div>
+
+      <div class="description">
+        <h1>Tonic</h1>
```
Comment (Kamba56) [inline index.html]: - [ ] Kindly use only one h1 tag in the whole of your HTML document. The h1 tag should be the main heading of your document. Change all other h1 tags to other heading tags `h2 ... h5`. This is best practice for SEO

File: index.html [hunk 3]
```diff
@@ -36,5 +36,190 @@
...
+        
+      </div>
+    </div>
+    </section>
+    <section class="about me">
+    <div class="third-page">
+       <div class="description">
+        <div class="paragraph">
+          <h2 id="title">About Myself</h2>
+          <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p></div>
```
Comment (Kamba56) [inline index.html]: - [ ] Kindly fix the layout of this p tag. it seems to be squeezed in the middle. Kindly increase the width to match the template.
- [ ] Kindly increase the size of the About section heading and reduce the space between the heading and the text.
### Your design
image
### Figma
image

File: index.html [hunk 4]
```diff
@@ -36,5 +36,190 @@
         </div>
       </section>
   </section>
+   <section class="work">
+    
+    
+    
+    <div class="second-page">
+      <div class="cards">
```
Comment (Kamba56) [inline index.html:44]: - [ ] As demanded by project requirements, kindly use CSS flexbox to layout contents of individual cards.
image

File: index.html [hunk 5]
```diff
@@ -36,5 +36,190 @@
...
+          </ul>
+        </span></div>
+        <div>
+        <button>see projects</button>
+        </div>
+        
+      </div>
+    </div>
+    </section>
+    <section class="about me">
```
Comment (Kamba56) [inline index.html:162]: - [ ] As demanded by project requirements, make sure the about section is laid out using CSS flexbox.
image

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @Etsubdink-Demeke, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job using flexbox to lay out the contents of the page!👏👏👏
- [x] Good job adding the work section and about section

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @Etsubdink-Demeke, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**